orm_lib/tests/db_test.cc

```diff
@@ -1930,5 +1943,17 @@ int main(int argc, char **argv)
 
     app().run();
     thr.join();
+
+    // Destruct the clients before event loop shutdown
+#if USE_MYSQL
+    mysqlClient = {};
+#endif
+#ifdef USE_POSTGRESQL
+    postgreClient = {};
+#endif
+#ifdef USE_SQLITE3
+    sqlite3Client = {};
+#endif
```

Comment:
Maybe it's better to use reset() method

Reply:
Will update it later (waiting for the CI result).

Should we consider this a bug? The SQLite3 client can crash itself during destruction if it holds a shared_ptr to itself and destructs inside it's own event loop

Reply:
Emm... it leads to the EventLoopThread is destoryed in its own thread, and the join method of std::thread throws an exception (join itself).
We could handle this later, it's not an emergency bug.

Reply:
Not quite. The abort is caused by `thread_.joinable()` being false due to the current thread is `thread_`. Causing the thread not being joined by anyone. Thus abort since the thread objects destructed without detaching or anyone joining it.